core/ledger/kvledger/txmgmt/statedb/statecouchdb/couchdb_test_export.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package statecouchdb
+
```

Comment:
The function and variable names in this file are a bit overloaded. For instance, cleanup v/s cleanupDB.

Can we use following three names
CouchDBSetup -> StartCouchDB 
cleanup -> stopCouchDBFunc
CleanupDB -> DeleteApplicationDBs